<map id="Mezzanine::NetworkPermissionException" name="Mezzanine::NetworkPermissionException">
<area shape="rect" id="node2" href="$classMezzanine_1_1NetworkException.html" title="Thrown when something unknown causes network IO to fail." alt="" coords="8,238,200,266"/>
<area shape="rect" id="node4" href="$classMezzanine_1_1IOException.html" title="Thrown when there was an issue with IO but very little was known about it." alt="" coords="25,161,183,189"/>
<area shape="rect" id="node6" href="$classMezzanine_1_1Exception.html" title="This is the exception thrown by most Mezzanine system that can throw exceptions." alt="" coords="32,83,176,111"/>
<area shape="rect" id="node8" title="STL class." alt="" coords="55,6,153,34"/>
</map>
